modules/accounts/email.nix

```
@@ -44,6 +44,13 @@ let
         '';
       };
 
+      command = mkOption {
+        type = with types; nullOr str;
```

```suggestion
        type = with types; nullOr (either str path);
```
This would allow using something like `pkgs.writeShellScript`